bench/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "bench"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+vuln-reach = { path = "../vuln-reach" }
+vuln-reach-master = { package = "vuln-reach", git = "ssh://github.com/phylum-dev/vuln-reach" }
```

```suggestion
vuln-reach-master = { package = "vuln-reach", git = "ssh://git@github.com/phylum-dev/vuln-reach" }
```
I believe `git@` should fix the CI issue.